### DISCUSSION TITLE
Welcome to AWS Copilot Discussions!

### DISCUSSION THREAD

**Discussion by kohidave** [aws/copilot-cli#1743] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by efekarakus** [comment]
Hi y'all! I'm one of the developers on the AWS Copilot team, I'm super excited to hear back from all of you :D

**Comment by miketheman** [comment]
Hey y'all! I'm an AWS Container Hero, and as a huge fan of ECS, I'm very excited for copilot to become the de-facto tool for building and deploying scalable application architectures with lower cognitive overhead.
